packages/java/endpoint/src/main/java/dev/hilla/BrowserCallable.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2000-2023 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package dev.hilla;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * Makes the methods of the annotated class available to the browser.
+ * <p>
+ * For each class, a corresponding TypeScript class is generated in
+ * {@code frontend/generated} with TypeScript methods for invoking the methods
+ * in this class.
```

Comment:
Shouldn't it be mentioned that this is an alias for the existing `@Endpoint` ? Otherwise, people are confused what's the difference...

Reply:
Added cross references